install/linux/usr/bin/odemis-start

```diff
@@ -225,6 +225,36 @@ class BackendStarter(object):
         self._notif.update(summary, message, icon)
         self._notif.show()
 
+    def proc_communicate(self, p):
+        """
+        Does the same as "out, err = p.communicate()", but (almost) immediately
+        returns when the process ends.
+        returns (list of strings, list of strings): out, err
+        """
+        def read_file(f, store):
```

Comment:
Do you read here the yaml file?

Reply:
Not the YAML file. Here, the word "file" is used in the generic UNIX sense: a byte output. It's actually used below only to read the special "pipes" stdout and stderr.